### PR TITLE
Support for zmpop and bzmpop commands

### PR DESCRIPTION
**Pull request by 72squared**

### CODE REVIEW

**Comment by 72squared** [comment]
Partially addresses issue #31

File: modules/api/src/main/scala/com/github/scoquelin/arugula/commands/RedisSortedSetAsyncCommands.scala [hunk 1]
```diff
@@ -16,34 +17,38 @@ trait RedisSortedSetAsyncCommands[K, V] {
...
   /**
-   * Remove and return the member with the highest score from one or more sorted sets
+   * Remove and return the member with the lowest score from one or more sorted sets
    * @param timeout The timeout
    * @param keys The keys
-   * @return The member with the highest score, or None if the sets are empty
+   * @return The member with the lowest score, or None if the sets are empty
    */
-  def bzPopMax(timeout: Long, keys: K*): Future[Option[ScoreWithKeyValue[K,V]]]
+  def bzPopMin(timeout: FiniteDuration, keys: K*): Future[Option[ScoreWithKeyValue[K,V]]]
```
Comment (72squared) [inline modules/api/src/main/scala/com/github/scoquelin/arugula/commands/RedisSortedSetAsyncCommands.scala:42]: I switched all the sorted set commands from specifying timeouts for blocking operations from longs and doubles to FiniteDuration. much clearer about intent and more consistent with how other Arugula commands are implemented.

File: modules/api/src/main/scala/com/github/scoquelin/arugula/commands/RedisSortedSetAsyncCommands.scala [hunk 2]
```diff
@@ -401,4 +423,11 @@ object RedisSortedSetAsyncCommands {
   final case class ScoreWithKeyValue[K, V](score: Double, key: K, value: V)
   final case class ZRange[T](start: T, end: T)
   final case class RangeLimit(offset: Long, count: Long)
+
+  sealed trait SortOrder
```
Comment (72squared) [inline modules/api/src/main/scala/com/github/scoquelin/arugula/commands/RedisSortedSetAsyncCommands.scala:427]: I didn't know what to call this enum. Direction didn't seem quite right to me.

Comment (scoquelin) [reply modules/api/src/main/scala/com/github/scoquelin/arugula/commands/RedisSortedSetAsyncCommands.scala:427]: `SortPriority` maybe ? Anyway, `SortOrder` works fine too!

**Review by scoquelin — APPROVED** [review]
Way more explicit with Duration 👍 Thanks !

**Pull request merged**